charts/dapr/charts/dapr_dashboard/values.yaml

```diff
@@ -0,0 +1,14 @@
+replicaCount: 1
+logLevel: info
+
+image:
+  name: dapr
+
```

Comment:
add a tag here as follows which can be referenced in dapr_dashboard_deployment.yaml 
tag: "0.1.0"